vcd_cli/network.py

```diff
@@ -676,6 +681,39 @@ def _get_platform(ctx):
     return Platform(client)
 
 
+@external.command(
+    'add-ip-range',
+    short_help='Adds an IP range of a subnet in an external network.')
```

Comment:
same as above use "to" instead of "of".
Naming as per confluence:
https://confluence.eng.vmware.com/pages/viewpage.action?spaceKey=VCD&title=Design+of+External+Network+commands+for+vcd-cli